pkg/services/featuremgmt/manager.go

```diff
@@ -184,3 +184,15 @@ func WithFeatures(spec ...interface{}) *FeatureManager {
 
 	return &FeatureManager{enabled: enabled}
 }
+
+// GetEnabledFlagNames returns a string array of enabled keys
+func (fm *FeatureManager) GetEnabledFlagNames() []string {
```

Comment:
Why not use GetEnabled directly to get the map, and just check the key exist in the map, it would be an operation of o(1) instead, except there is a special needs here, I don't think the method is needed.